src/Facade/ResourceBuilders/SalesArticlesReallocatorResourceBuilder.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Linn.Products.Facade.ResourceBuilders
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Linn.Common.Facade;
+    using Linn.Common.Resources;
+    using Linn.Products.Domain.Linnapps;
+    using Linn.Products.Resources;
+
+    public class SalesArticlesReallocatorResourceBuilder : IResourceBuilder<ResponseModel<SalesArticlesReallocator>>
+    {
+        public SalesArticlesReallocatorResource Build(ResponseModel<SalesArticlesReallocator> salesArticlesReallocator)
+        {
+            return new SalesArticlesReallocatorResource
+            {
+                OldTariffId = salesArticlesReallocator.ResponseData.OldTariffId.ToString(),
+                NewTariffId = salesArticlesReallocator.ResponseData.NewTariffId.ToString(),
```

Comment:
why are you making these strings out of interest?

Reply:
Ah good point, they could all be ints now I think. I initially made a mistake and was using the wrong id which was a string, I seem to have got confused and left some as strings